chapter1.md

```diff
@@ -73,48 +71,48 @@ Figure 1 shows three example sites with the empirical distribution and associate
 
-![Number of cases where each model provided the best fit to the data based on AICc for each dataset separately.](./wins-by-dataset.png)
+![Number of cases in which each model provided the best fit to the data based on AICc for each dataset separately.](./wins-by-dataset.png)
 
 The full distribution of AICc weights shows separation among models (Figure 5). Although the log-series distribution had the best AICc score much more often than the other models, its lead was never decisive: across all 16,209 sites, it never had more than about 75% of the AICc weight (Figure 5). Most of the remaining weight was assigned to the negative binomial and Poisson lognormal distributions (each of which usually had at least 12-15% of the weight but was occasionally favored very strongly). The Zipf distribution showed a strong mode near zero, and usually had less than 7% of the weight.
 
 ![Violin plots of the AICc weights for each model. Weights indicate the probability that the model is the best model for the data](./violins/weight.png)
 
 # Discussion
 
-Our extensive comparison of different models for the species abundance distribution (SAD) using rigorous statistical methods demonstrates that several of the most popular existing models provide equivalently good absolute fits to empirical data. Log-series, negative binomial, and Poisson lognormal all had model relative likelihoods between 0.25 and 0.5 suggesting that the three distributions provide roughly equivalent fits in most cases, but with the two parameters models doing slightly better on average. Because the log-series has only a single parameter but fits the data almost as well as the two parameter models, the log-series performed better in AICc-based model selection, which penalizes model complexity. These results differ from two other recent analyses of large numbers of species abundance distributions [@ulrich2010; @connolly2014] and are generally consistent with a third recent analysis [@alroy2015].
+Our extensive comparison of different models for the species abundance distribution (SAD) using rigorous statistical methods demonstrates that several of the most popular existing models provide equivalently good absolute fits to empirical data. Log-series, negative binomial, and Poisson lognormal all had model relative likelihoods between 0.25 and 0.5 suggesting that the three distributions provide roughly equivalent fits in most cases, but with the two parameter-model performing slightly better on average. Because the log-series has only a single parameter but fits the data almost as well as the two parameter models, the log-series performed better in AICc-based model selection, which penalizes model complexity. These results differ from two other recent analyses of large numbers of species abundance distributions [@ulrich2010; @connolly2014] and are generally consistent with a third recent analysis [@alroy2015].
```

Comment:
change "two parameter models" to "two-parameter models" because we have models with two parameters, not two models that have parameters.